modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -0,0 +1,63 @@
+[[troubleshooting-clients]]
+= Troubleshooting Clients
+// Antora entities
+ifndef::backend-pdf[]
+include::ROOT:partial$entities.adoc[]
+endif::[]
+
+// PDF entities
+ifdef::backend-pdf[]
+include::_partials/entities.adoc[]
+endif::[]
+
+
+// Keep this in alphabetical order. We might also want to consider breaking these into their own topics, like we have in the admin guide. LKB 2019-04-05
+
+== Cloned Salt Clients
+
+If you have used your hypervisor clone utility, and attempted to register the cloned Salt client, you might get this error:
+
+----
+We're sorry, but the system could not be found.
+----
+
+This is caused by the new, cloned, system having the same machine ID as an existing, registered, system.
+You can adjust this manually to correct the error and register the cloned system successfully.
+
+
+For more information and instructions, see pass:c[xref:FILENAME.adoc#bp.chapt.suma3.troubleshooting.registering.cloned.salt.systems[]].
+
+
+
+== Mounting /tmp with noexec
+
+Salt runs remote commands from [filename]``/tmp`` of the client's filesystem.
+Therefore you must not mount [filename]``/tmp`` with the [option]``noexec`` option.
+
+
+
+== SSL errors
+
+On {slsa}{nbsp}11 systems, clients can sometimes have SSL errors which make some operations unusable, including package management and bootstrapping.
+In this case, you will see an error like this:
+
+----
+Repository 'SLES11-SP4-SUSE-Manager-Tools x86_64' is invalid.
+[|] Valid metadata not found at specified URL(s)
+Please check if the URIs defined for this repository are pointing to a valid repository.
+Skipping repository 'SLES11-SP4-SUSE-Manager-Tools x86_64' because of the above error.
+Download (curl) error for 'www.example.com':
+Error code: Unrecognized error
+Error message: error:1409442E:SSL routines:SSL3_READ_BYTES:tlsv1 alert protocol version
+----
+
+This occurs because Apache requires TLS{nbsp}v1.2, but older versions of {slsa} do not support this version of the TLS protocol.
```

Comment:
RHEL 6 might have the same problem. Maybe QA can test it when there is a bit of time after Beta 2 is out.

Reply:
We can always update if they find new info.  I'll merge this now.